index.html

```diff
@@ -1194,16 +1197,102 @@
             This section is currently work in progress, and subject to change.
           </div>
           <p>
+            Use cases like rendering 3D point cloud, background removal,
```

Comment:
I would do

There are several use-cases which are a good fit to be, at least partially, implemented on the GPU, such as motion recognition, pattern recognition, background removal, as well as 3D point cloud.

This section explains which APIs can be used for some of these mentioned use-cases; ...